client/client.go

```diff
@@ -923,16 +926,28 @@ func (r *NotaryRepository) RotateKey(role string, serverManagesKey bool) error {
 	}
 
 	var (
-		pubKey    data.PublicKey
-		err       error
-		errFmtMsg string
+		pubKey     data.PublicKey
+		pubKeyList data.KeyList
+		err        error
+		errFmtMsg  string
 	)
-	switch serverManagesKey {
-	case true:
+
+	switch {
```

Comment:
Hi @ecordell it's just a style preference, personally :) 
Do you thinks this works for you,
```
        if serverManagesKey {
                pubKey, err = rotateRemoteKey(r.baseURL, r.gun, role, r.roundTrip)
                pubKeyList = make(data.KeyList, 0, 1)
                pubKeyList = append(pubKeyList, pubKey)
                errFmtMsg = "unable to rotate remote key: %s"
        } else {
                if len(keyList) == 0 { 
                        pubKeyList = make(data.KeyList, 0, 1)
                        pubKey, err = r.CryptoService.Create(role, r.gun, data.ECDSAKey)
                        pubKeyList = append(pubKeyList, pubKey)
                        errFmtMsg = "unable to generate key: %s"
                } else {
                        pubKeyList = make(data.KeyList, 0, len(keyList))
                        for _, keyID := range keyList {
                                pubKey = r.CryptoService.GetKey(keyID)
                                pubKeyList = append(pubKeyList, pubKey)
                        }   
                }   
        }
```
 but feel free to keep your code style.